index.ts

```diff
@@ -118,6 +201,107 @@ export default class CountTransformer
       stack
     );
 
+    for (const config of this.fields) {
+      const {
+        countFields,
+        field,
+        fields,
+        object,
+        resolverTypeName,
+        resolverFieldName,
+      } = config;
+
+      const localFields = fields.length > 0 ? fields : countFields;
+
+      // Find the table we want to scan
+      const tableDataSource = ctx.dataSources.get(
+        resolverTypeName
+      ) as appsync.DynamoDbDataSource;
+      const table = tableDataSource.ds
+        .dynamoDbConfig as appsync.CfnDataSource.DynamoDBConfigProperty;
+
+      const dataSource = ctx.api.host.getDataSource(
+        `${resolverTypeName.name.value}Table`
+      );
+
+      // Allow the lambda to access this table
+      funcRole.addToPolicy(
+        new iam.PolicyStatement({
+          actions: ["dynamodb:Scan"],
+          effect: iam.Effect.ALLOW,
+          resources: [
+            `arn:aws:dynamodb:${table.awsRegion}:${stack.account}:table/${table.tableName}`,
+          ],
+        })
+      );
+
+      // Create the GraphQL resolvers.
+      const resolverId = ResolverResourceIDs.ResolverResourceID(
+        config.resolverTypeName.name.value,
+        config.resolverFieldName
+      );
+      let resolver = createdResources.get(resolverId);
+
+      const requestTemplate: Array<Expression> = [
+        qref(`$ctx.stash.put("typeName", "${config.resolverTypeName}")`),
+        qref(`$ctx.stash.put("fieldName", "${config.resolverFieldName}")`),
+      ];
+
+      const authModes = [
```

Comment:
Can you import this code from the `auth` module? I assume it's copied from there.